frontend/src/individual_certificate.tsx

```diff
@@ -115,7 +115,13 @@ const useStyles = makeStyles({
   },
 });
 
-export function IndividualCertificate({ id }: { id: string }) {
+export function IndividualCertificate({
+  id,
+  action,
+}: {
+  id: string;
+  action: string;
```

Comment:
I have a slight preference for using the language strengths and going for an `enum` here.